extensions/wikia/FlowTracking/FlowTracking.hooks.php

```diff
@@ -55,6 +75,33 @@ public static function getParamsFromUrlQuery( $url ) {
 		return $queryParams;
 	}
 
+	/**
+	 * @param PageHeaderController $header
+	 * @param $actions
+	 * @return bool
+	 */
+	public static function onBeforePrepareActionButtons( $header, &$actions ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameter "$header". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1172)